risc0/zkvm/src/host/protos/core.proto

```diff
@@ -47,13 +47,14 @@ message SegmentReceipt {
   bytes seal = 2;
   uint32 index = 3;
   string hashfn = 4;
+  ReceiptMetadata metadata = 5;
 }
 
 message SuccinctReceipt {
   protos.base.CompatVersion version = 1;
   bytes seal = 2;
   Digest control_id = 3;
-  ReceiptMetadata meta = 4;
+  ReceiptMetadata metadata = 4;
```

Comment:
I don't think this is an issue now, but for future reference, if we were using JSON, this would be considered a breaking change.

Reply:
Oh, that's a good point 😅